docs/source/release_notes.rst

```diff
@@ -24,6 +24,8 @@ Future Release
         * Allow deep EntitySet equality check (:pr:`1480`)
         * Update ``EntitySet.concat`` to work with Woodwork DataFrames (:pr:`1490`)
         * Add function to list semantic tags (:pr:`1486`)
+        * Remove categorical-encoding as an add-on library (:pr:`1632`)
```

Comment:
Unrelated?

Reply:
oops copied accidentally in a merge conflict. Thanks!